src/Opdex.Platform.WebApi/Models/Requests/FilterParameters.cs

```diff
@@ -19,12 +21,16 @@ public abstract class FilterParameters<TCursor> where TCursor : Cursor
         /// <summary>
         /// Number of results to return per page.
         /// </summary>
-        public uint Limit { get; set; }
+        // virtual is used to be able to override the attribute
+        [Range(1, Cursor.DefaultMaxLimit)]
+        public virtual uint Limit { get; set; }
 
         /// <summary>
         /// The cursor when paging.
         /// </summary>
-        public string Cursor { get; set; }
+        [NotNull]
+        [FromQuery(Name = "Cursor")]
+        public string EncodedCursor { get; set; }
```

Comment:
We could probably use a base class here for these filter requests to mitigate the repetitiveness and going forward take care of the shared properties in one place.

Reply:
Also the cursor can be not null

Reply:
Not sure I understand? This class is a base class for those properties

Reply:
👀 ....... 💀